ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -229,13 +229,16 @@ initNodeKernel args@NodeKernelArgs { registry, cfg, tracers
               , GSM.setCaughtUpPersistentMark = \upd ->
                   (if upd then GSM.touchMarkerFile else GSM.removeMarkerFile)
                     gsmMarkerFileView
-              , GSM.writeLedgerStateJudgement   = \x -> atomically $ do
-                  writeTVar varLedgerJudgement x
+              , GSM.writeGsmState = \x -> atomically $ do
+                  writeTVar varLedgerJudgement $ GSM.gsmStateToLedgerJudgement x
+              , -- In the context of bootstrap peers, it is fine to always
```

Comment:
Will this be the appropriate place to constrain the Diffusion Layer output require by https://github.com/IntersectMBO/ouroboros-network/issues/4815 once that's available?

Reply:
Yeah, I think that makes sense, basically a particular interpretation of what "honest availability" means in the context of bootstrap peers :+1: